site/en/tutorials/audio/simple_audio.ipynb

```diff
@@ -99,7 +99,7 @@
       "outputs": [],
       "source": [
         "!pip install -U -q tensorflow tensorflow_datasets\n",
-        "!apt install --allow-change-held-packages libcudnn8=8.1.0.77-1+cuda11.2"
+        "!apt install --allow-change-held-packages libcudnn8=8.6.0.163-1+cuda11.8"
```

Comment:
```suggestion
```

@MarkDaoust 
Ran the notebook with GPU enabled without this line. Let's remove it.

And thank you @chunduriv